docs/workflows/sign-images.rst

```diff
@@ -0,0 +1,301 @@
+.. _sign-images:
+
+Image signature configuration
+==============================
+
+Administrators can add container manifest signing service to Pulp Registry using the command line
+tools. Users may then associate the signing service with container repositories.
+The example below demonstrates how manifest signing service can be created using ``gpg``:
+
+1. Make sure the service user ``pulp`` has access to ``gpg`` and that the keypair is
+   installed in its keyrings. The private key might alternatively be provided by a
+   hardware cryptographic device.
+
+2. Create a signing script that accepts manifest path as the only argument. The script invokes
+   ``skopeo standalone-sign`` command that generates container signature for the image manifest,
+   using the key specified via the ``PULP_SIGNING_KEY_FINGERPRINT`` environment variable. The script
+   should then print out a JSON structure with the following format. Signature path for the created
+   signature is relative path inside the current working directory::
+
+       {"signature_path": "signature"}
+
+   Below is an example of a signing script:
+
+   .. code-block:: bash
+
+       #!/usr/bin/env bash
+
+        MANIFEST_PATH1=$1
+        FINGEPRINT="$PULP_SIGNING_KEY_FINGERPRINT"
+        REFERENCE="$IMAGE_REFERENCE"
+        SIG_PATH="$SIGNATURE_PATH"
+
+        # Create container signature
+        skopeo standalone-sign $MANIFEST_PATH $REFERENCE $FINGEPRINT --output $SIG_PATH
+        # Check the exit status
+        STATUS=$?
+        if [ $STATUS -eq 0 ]; then
+          echo {\"signature_path\": \"$SIG_PATH\"
+        else
+          exit $STATUS
+        fi
+
+   .. note::
+
+       Make sure the script contains a proper shebang and Pulp has got valid permissions
+       to execute it.
+       Since the script invokes a ``skopeo`` command, it should be installed as well.
+
+3. Create a signing service consisting of an absolute path to the script and a meaningful
+   name describing the script's purpose. It is possible to insert the signing service in
+   to a database by using the ``pulpcore-manager shell_plus`` interactive Python shell, however
```

Comment:
How about sharing only the add-signing-service way? Is there any limitation to it so users will need to use django shell?

If we need to mention the django shell, let's share the `pulpcore-manager shell` because it is available for any installation. `shell_plus` comes from the extensions which are not installed by default and some users might not have access to install it.

Reply:
in core we advertise both so i just followed same style, +1 to mention only add-signing-service